DetectorDescription/DDCMS/interface/DDShapes.h

```diff
@@ -0,0 +1,242 @@
+#ifndef Detector_Description_DDCMS_DDShapes_h
+#define Detector_Description_DDCMS_DDShapes_h
+
+#include "DetectorDescription/DDCMS/interface/DDFilteredView.h"
+
+// When old DD is eliminated, remove this include
+#include "DetectorDescription/Core/interface/DDSolidShapes.h"
+
+namespace cms {
+  namespace dd {
+
+    // When old DD is eliminated, uncomment this enum
+    /*
+      enum class DDSolidShape {
+        dd_not_init = 0,
+        ddbox = 1,
+        ddtubs = 2,
+        ddtrap = 3,
+        ddcons = 4,
+        ddpolycone_rz = 5,
+        ddpolyhedra_rz = 6,
+        ddpolycone_rrz = 7,
+        ddpolyhedra_rrz = 8,
+        ddtorus = 9,
+        ddunion = 10,
+        ddsubtraction = 11,
+        ddintersection = 12,
+        ddshapeless = 13,
+        ddpseudotrap = 14,
+        ddtrunctubs = 15,
+        ddsphere = 16,
+        ddellipticaltube = 17,
+        ddcuttubs = 18,
+        ddextrudedpolygon = 19,
+      };
+      */
+
+    DDSolidShape getCurrentShape(const cms::DDFilteredView &fview);
```

Comment:
@ianna: A while ago, I think you said you did not want `DDSolidShape` in the `DDFilteredView`. If you've changed your mind, I can move this method.
Actually, I think it is cleaner to limiting this dependency on the old DD `DDSolidShape` to `DDShapes.h`, rather than adding it to the more widely used `DDFilteredView.h`.
What do you think?

Reply:
I still think we do not need ```DDSolidShape```. However, if it's there I'd rather have it in ```DDFilteredView```. The problem with having it outside is that even if the reference is declared const, the current shape may change.

Reply:
@ianna so do we postpone this update to a future PR?